lighthouse-core/gather/driver.js

```diff
@@ -24,6 +25,21 @@ const pageFunctions = require('../lib/page-functions.js');
 // eslint-disable-next-line no-unused-vars
 const Connection = require('./connections/connection.js');
 
+const UIStrings = {
+  /**
+   * @description Warning that important data was not cleared but may have affected the scores of this run.
```

Comment:
something like

```suggestion
   * @description A warning that previously-saved data may have affected the measured performance and instructions on how to avoid the problem. "locations" will be a list of possible types of data storage locations, e.g. "IndexedDB",  "Local Storage", or "Web SQL".
```

?

Reply:
+ "these locations will not be translated".